dataactbroker/fileRoutes.py

```diff
@@ -130,6 +130,70 @@ def get_protected_files():
         file_manager = FileHandler(request, is_local=is_local, server_path=server_path)
         return file_manager.get_protected_files()
 
+    @app.route("/v1/check_current_page/", methods=["GET"])
+    @convert_to_submission_id
```

Comment:
Requiring login doesn't ensure they have at least read only permissions. It essentially is just saying you can log into MAX and get to the broker successfully. I would suggest making this require _at least_ readonly permissions.